draft-ietf-httpbis-digest-headers.md

```diff
@@ -1263,6 +1267,15 @@ This memo sets this specification to be the establishing document for the [HTTP
 Algorithm
 Values](https://www.iana.org/assignments/http-dig-alg/) registry.
 
+IANA is asked to update the "Reference" for this registry
+to refer this document
+and to inizialize the registry with the tokens 
+defined in {{algorithms}}.
+
+New registrations will be permitted
+through the RFC Required policy (see Section 4.7 of {{!RFC8126}}).
```

Comment:
as per my comment on the issue, I think specification required is fine. So the suggestion here is

```suggestion
Registrations or this registry use the Specification
Required policy (Section 4.6 of {{!RFC8126}}).
The designated expert or experts verify that a
specification exists and is readily accessible.
```

Reply:
+1 to Specification Required. You could also shorten it to "This registry use the Specification
Required policy (Section 4.6 of {{!RFC8126}}).

Reply:
In case of "Specification Required", how to ensure interoperability with the overall HTTP work if the Specification is published outside IETF (eg. W3C, OASIS, ...) ? 

Further notes are here https://github.com/httpwg/http-extensions/issues/1533#issuecomment-916462729

Reply:
You don't.  It might be nice in theory to insist on people only deploying things that have been carefully designed and vetted, but that doesn't happen.  If you make the requirements on registrations very difficult to achieve, what you end up with is people not registering things.  They will still use the codepoints and deploy the code, so what you end up with is no reliable record of what codepoints are safe to use.  Even "Specification Required" can be too much to avoid that happening, but we should at least encourage people to write down what they are doing.